### ISSUE TITLE
Add "custom" random Ai

### ISSUE THREAD

**Issue by Ffffffmaa** [celguar/mangosbot-bots#203]
I was wondering, and after repeatedly searching haven't found a clear answer. Is there a way to add "custom" random ai... Meaning a random ai with race + gender + class + name? (for example, i wish to add some characters with characteristics my old wow friends had.)
Also, is there any command to make a random ai to abandon it's guild?

Thank you

**Comment by celguar** [comment]
Hello, atm there is no command to create a random bot of desired parameters. But frankly sounds like a good idea. For now you can make these chars in your account and summon them as bots

For guild try invite bot and whisper it "guild leave" or "do guild leave"

**Comment by Ffffffmaa** [comment]
Thank you

**Comment by Ffffffmaa** [comment]
Yes, i have a secondary account, but that contains my horde characters, and yes, while the "main" contains the alliance characters, i do sometimes run both accounts to face them off at some "world pvp spot".

 Although, i was wondering if for the moment, "capturing" random ai and rename them through gm commands could impact them, like breaking them or the db.

**Comment by kadeshar** [comment]
You tried this commands? For me bots rememeber their builds even after server reset.

/w talents  - this command is for checking actual bot talents build
/w talents list - this command is for available builds for bot (based on config file. You can add your custom build)
/w talents build_name_from_list - this command changing actual build on specified name

**Comment by Ffffffmaa** [comment]
I will have to try the /w talent list next time i'm on,the problem i have is that, how i explain, it doesn't reset when server restarts, it suddenly resets talents and gear configurations, i ended up throwing up all other gear she had in the inventory, cause once she had starting character lvl 1 gear equipped and tier 6 in the bag.... so i decided to have her and the other ones that behave that way, with no other gear in the inventory and check up their talents every now and then.

**Comment by mostlikely4r** [comment]
Bots have their (hidden) internal talent path they want to follow. If you log into their char and change their talents what should they pick when they level up? So they move back to their own idea.

'talents name_of_config_build/wowheadlink' changes their idea and makes them stick to it.

**Comment by mostlikely4r** [comment]
"Although, i was wondering if for the moment, "capturing" random ai and rename them through gm commands could impact them, like breaking them or the db."

Renaming bots should not affect them. All players have internal game ID used to logic. Name is just a label.

But try not to have 2 players with same name. That might cause small issues with /invite or /who

**Comment by Ffffffmaa** [comment]
> "Although, i was wondering if for the moment, "capturing" random ai and rename them through gm commands could impact them, like breaking them or the db."
> 
> Renaming bots should not affect them. All players have internal game ID used to logic. Name is just a label.
> 
> But try not to have 2 players with same name. That might cause small issues with /invite or /who

Ty for the answer, sorry for not updating with the "work around" i found, as suggested above by @celguar , i did create another account with the desired characters, initially i had 2 instances of tbc running, but one day i tried activating those characters through ".bot add xxxx" and it turns out it does enable them and they work as intended without needing to have a second wow process running. 

As for teh talents, thank you for the suggestion, i will have to try it the next time i have some free time for it, thank you for the answers.
